app/assets/images/discord.svg

```diff
@@ -0,0 +1,6 @@
+<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="24pt" height="24pt" viewBox="0 0 24 24" version="1.1">
+<g id="surface1">
+<path style=" stroke:none;fill-rule:nonzero;fill:rgb(100%,100%,100%);fill-opacity:1;" d="M 10.226562 10.390625 C 9.667969 10.390625 9.226562 10.890625 9.226562 11.5 C 9.226562 12.109375 9.679688 12.609375 10.226562 12.609375 C 10.785156 12.609375 11.226562 12.109375 11.226562 11.5 C 11.234375 10.890625 10.785156 10.390625 10.226562 10.390625 Z M 13.800781 10.390625 C 13.242188 10.390625 12.804688 10.890625 12.804688 11.5 C 12.804688 12.109375 13.253906 12.609375 13.800781 12.609375 C 14.359375 12.609375 14.800781 12.109375 14.800781 11.5 C 14.800781 10.890625 14.359375 10.390625 13.800781 10.390625 Z M 13.800781 10.390625 "/>
+<path style=" stroke:none;fill-rule:nonzero;fill:rgb(100%,100%,100%);fill-opacity:1;" d="M 18.5625 2 L 5.4375 2 C 4.328125 2 3.429688 2.921875 3.429688 4.058594 L 3.429688 17.578125 C 3.429688 18.71875 4.328125 19.640625 5.4375 19.640625 L 16.546875 19.640625 L 16.027344 17.789062 L 17.28125 18.980469 L 18.464844 20.101562 L 20.570312 22 L 20.570312 4.058594 C 20.570312 2.921875 19.671875 2 18.5625 2 Z M 14.78125 15.058594 C 14.78125 15.058594 14.429688 14.628906 14.136719 14.25 C 15.417969 13.878906 15.910156 13.058594 15.910156 13.058594 C 15.507812 13.328125 15.125 13.519531 14.78125 13.648438 C 14.292969 13.859375 13.820312 14 13.363281 14.078125 C 12.421875 14.261719 11.558594 14.210938 10.824219 14.070312 C 10.265625 13.960938 9.785156 13.800781 9.382812 13.640625 C 9.160156 13.550781 8.914062 13.441406 8.667969 13.300781 C 8.640625 13.28125 8.609375 13.269531 8.582031 13.25 C 8.5625 13.238281 8.550781 13.230469 8.542969 13.21875 C 8.367188 13.121094 8.269531 13.050781 8.269531 13.050781 C 8.269531 13.050781 8.738281 13.851562 9.980469 14.230469 C 9.6875 14.609375 9.324219 15.058594 9.324219 15.058594 C 7.160156 14.988281 6.339844 13.539062 6.339844 13.539062 C 6.339844 10.320312 7.75 7.710938 7.75 7.710938 C 9.160156 6.628906 10.5 6.660156 10.5 6.660156 L 10.597656 6.78125 C 8.835938 7.300781 8.023438 8.089844 8.023438 8.089844 C 8.023438 8.089844 8.238281 7.96875 8.601562 7.800781 C 9.648438 7.328125 10.480469 7.199219 10.824219 7.171875 C 10.882812 7.160156 10.933594 7.148438 10.992188 7.148438 C 11.589844 7.070312 12.265625 7.050781 12.96875 7.128906 C 13.902344 7.238281 14.898438 7.519531 15.917969 8.089844 C 15.917969 8.089844 15.144531 7.339844 13.480469 6.820312 L 13.617188 6.660156 C 13.617188 6.660156 14.957031 6.628906 16.367188 7.710938 C 16.367188 7.710938 17.78125 10.320312 17.78125 13.539062 C 17.78125 13.539062 16.945312 14.988281 14.78125 15.058594 Z M 14.78125 15.058594 "/>
+</g>
+</svg>
```

Comment:
By default we try to use icons from remixicon.com (open source library of icons) with 24x24px size.
Also, lets make sure these are properly optimized with, for example, https://jakearchibald.github.io/svgomg/

Reply:
thanks @ludwiczakpawel 
I added replaced the discord.svg with the one from remixicon in this commit cb714a7